### ISSUE TITLE
Creating ghost roles is a little buggy

### ISSUE THREAD

**Issue by AJCM-git** [space-wizards/space-station-14#3189]
When you want to make a ghost role, you get the following exception:
```
[WARN] con: localhost@JoeGenero: ExecuteError - addcomp 1216 GhostRoleMobSpawner:
System.InvalidOperationException: Component reference type Content.Server.GameObjects.Components.Observer.GhostRoleComponent already occupied by Content.Server.GameObjects.Components.Observer.GhostTakeoverAvailableComponent
   at Robust.Shared.GameObjects.ComponentManager.AddComponent[T](IEntity entity, T component, Boolean overwrite) in /home/maharba/Save/3-Programas/Git/Desarrollo/SS14/space-station-14/RobustToolbox/Robust.Shared/GameObjects/ComponentManager.cs:line 99
   at Robust.Server.Console.Commands.AddCompCommand.Execute(IConsoleShell shell, String argStr, String[] args) in /home/maharba/Save/3-Programas/Git/Desarrollo/SS14/space-station-14/RobustToolbox/Robust.Server/Console/Commands/AddCompCommand.cs:line 35
   at Robust.Server.Console.ServerConsoleHost.ExecuteInShell(IConsoleShell shell, String command) in /home/maharba/Save/3-Programas/Git/Desarrollo/SS14/space-station-14/RobustToolbox/Robust.Server/Console/ServerConsoleHost.cs:line 96
```
 when you either add `GhostTakeOverAvaible` or `GhostRoleMobSpawn` components, the first one that you select stays in the VV list and the second one you kinda cant add because of the exception above (you can add GhostTakeOverAvaible first and then GhostRoleMobSpawn but in this case you cant edit the ghost role)
 @Zumorica

**Comment by Zumorica** [comment]
This is because `GhostTakeoverAvailable` and `GhostRoleMobSpawn` both inherit `GhostRole`, and only one of them can be active on a given entity at a time. If you read this exception, it VERY CLEARLY tells you. `Component reference type Content.Server.GameObjects.Components.Observer.GhostRoleComponent already occupied by Content.Server.GameObjects.Components.Observer.GhostTakeoverAvailableComponent`

TL;DR This is not a bug, this is an admin fuckup
